homeassistant/components/media_finder/__init__.py

```diff
@@ -0,0 +1,156 @@
+"""The media_finder integration."""
```

Comment:
This should be a helper instead of its own integration. Maybe even be part of the `media_player` integration (`media_palyer/media_finder.py`).

I love your split btw, as this means that now any media player integration that wants to support local can just offer a local folder in the initial list and it can just be routed through this helper.

Reply:
It would not need its own websocket commands.

Reply:
Oooh Bram even had an even more brilliant plan. 

If we know a media player can play arbitrary urls, we can offer moure "sources" than just local. One could make an MP3 player source or whatever. 

So in that case, maybe we should keep it an integration so we can allow other integrations to hook into this. Anything that has playable media can then offer sources.

Reply:
I like the idea of it being a helper or part of core vs an integration, but I'm not sure I fully follow the architecture for the "sources" concept. The websocket command is there really for the development of a card that doesn't use the media browser interface. A card that plays local media won't really be a media player per say since the device is the current browser and not external. That card could hit the new websocket command. There could be a cleaner way though...